src/config.rs

```diff
@@ -429,11 +435,14 @@ pub struct WdlStorageConfig {
 
 // Function for providing the default value
 fn wdl_directory_default() -> String {
-    let mut current_dir = std::env::current_dir()
-        .expect("Failed to get current directory for wdl directory default");
+    let mut current_dir =
+        std::env::current_dir().expect("Failed to get current directory for wdl directory default");
     current_dir.push("carrot");
     current_dir.push("wdl");
-    current_dir.to_str().expect("Failed to convert wdl directory path to string").to_string()
+    current_dir
+        .to_str()
```

Comment:
what's with the double to_string?

Reply:
`to_str()` returns a `&str`, which is a reference to the path in `current_dir`, interpreted as UTF-8.  The function can't return that because it's referencing data in a variable that goes out of scope at the end of that function, so `to_string()` copies the data into a `String`, which can be returned.